docs/api.md

```diff
@@ -29,9 +29,10 @@
 
 ## Events
 
-| Event             | Type                                  |
-|-------------------|---------------------------------------|
-| `dropdownToggled` | `CustomEvent<{ expanded: boolean; }>` |
+| Event                        | Type                                  | Description                                 |
+|------------------------------|---------------------------------------|---------------------------------------------|
+| `auroDropdown__triggerClick` | `CustomEvent<any>`                    | Notifies that the trigger has been clicked. |
```

Comment:
As mentioned in another PR, I'm not a fan of this event naming pattern. For consistency with the other event, it should be `triggerClick`. `auroDropdown` is redundant since you would listen to the event on an `<auro-dropdown>` element.

I'd also consider something like `auro-triggerclick`, but that would be inconsistent with the existing `dropdownToggled` event.